central/detection/alertmanager/alert_manager_impl.go

```diff
@@ -139,7 +139,7 @@ func (d *alertManagerImpl) shouldDebounceNotification(ctx context.Context, alert
 		AddExactMatches(search.PolicyID, alert.GetPolicy().GetId()).
 		AddExactMatches(search.ViolationState, storage.ViolationState_RESOLVED.String()).
 		ProtoQuery()
-	resolvedAlerts, err := d.alerts.SearchRawAlerts(ctx, q)
+	resolvedAlerts, err := d.alerts.SearchRawAlerts(ctx, q, true)
```

Comment:
I think for correctness, this one should be false.  The query is specifically looking for Resolved.  Pretty odd if we set the flag to exclude them and then ignore it because the query overwrote it.